helm-charts/falcon-sensor/values.yaml

```diff
@@ -15,6 +15,14 @@ node:
 
     updateStrategy: RollingUpdate
 
+    tolerations:
+    # this toleration is to have the daemonset runnable on master nodes
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+
+    nodeSelector:
+      beta.kubernetes.io/os: linux
```

Comment:
still depreacted, kubernetes.io/os is the new stable selector

Reply:
FYI @uncycler

Reply:
I think new rebase is need after the release today. We also wait on this to set also "kubernetes.io/arch: amd64".
Looks like no ARM64 images or multiarch image for Linux is available yet.